### PR TITLE
pulpcore bindings update for pulpcore 3.3

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by jlsherrill** [comment]
require https://github.com/theforeman/foreman-packaging/pull/5160

**Comment by evgeni** [comment]
shouldn't you also bump the buildrequires?

File: packages/katello/rubygem-katello/rubygem-katello.spec [hunk 1]
```diff
@@ -234,6 +234,9 @@ cp -pa .%{gem_dir}/* \
 %{gem_instdir}/webpack
 
 %changelog
+* Mon May 04 2020 Justin Sherrill <jsherril@redhat.com> 3.16.0-0.12.pre.master
+Update to pulp 3.3 bindings
```
Comment (evgeni) [inline packages/katello/rubygem-katello/rubygem-katello.spec:238]:
```suggestion
- Update to pulp 3.3 bindings
```

**Comment by zjhuntin** [comment]
[test rpm]

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
```
ModuleNotFoundError: Module not found: Error: Can't resolve '@patternfly/react-icons' in '/builddir/build/BUILDROOT/rubygem-katello-3.16.0-0.12.pre.master.20200507163221git2adce62.el8.noarch/usr/share/gems/gems/katello-3.16.0.pre.master/webpack/scenes/ContentViews/Table'
```

🤬

**Comment by evgeni** [comment]
@johnpmitsch is that error related to https://github.com/Katello/katello/commit/44f712016f8c6c873b7daec4f158dc2a0bd9b48a maybe?

**Comment by johnpmitsch** [comment]
@evgeni most likely, that is where the package was added. What can I update?

**Comment by evgeni** [comment]
repoclosure expected, thanks @jlsherrill!

**Pull request merged**